src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+

```

Comment:
Remove this unnecessary empty line to maintain a clean and organized code structure.